src/test/py/bazel/bzlmod/bazel_fetch_test.py

```diff
@@ -29,6 +29,7 @@ def setUp(self):
     self.main_registry = BazelRegistry(
         os.path.join(self.registries_work_dir, 'main')
     )
+    self.main_registry.start()
```

Comment:
Is there a way to adapt a `with`-able object for use in `setUp`/`tearDown` style function?